modules/oadp-1-3-restoring-csi-snapshots.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp_built_in_data_mover/oadp-backup-restore-csi-snapshots.adoc           
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-1-3-restoring-csi-snapshots_{context}"]
+= Restoring CSI volume snapshots
+
+You can restore a volume snapshot by creating a `Restore` CR.
+
+[NOTE]
+====
+You cannot restore Volsync backups from OADP 1.2 with the OAPD 1.3 built-in Data Mover. It is recommended to do a file system backup of all of your workloads with Restic prior to upgrading to OADP 1.3. 
+====
+
+.Prerequisites
+
+* You have access to the cluster with the `cluster-admin` role.
+* You have an OADP `Backup` CR from which to restore the data.
+
+.Procedure
+
+. Create a YAML file for the `Restore`  CR, as in the following example:
++
+.Example `Restore` CR
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore
+  namespace: openshift-adp
+spec:
+  backupName: <backup> 
+  restorePVs: true 
```

Comment:
Maybe we can remove the restorePVs flag from restore CR example. if this field is empty or not specified then by default it will restore all the PV's